src/engine/ruleset/decoders/fortinet/fortimanager.yml

```diff
@@ -0,0 +1,201 @@
+---
+name: decoder/fortinet-fortimanager/0
+
+metadata:
+  title: Decoder for Fortinet FortiManager Endpoint Security
+  description: Decoder for Fortinet FortiManager Endpoint Security
+  compatibility: >
+    Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/20
+  references:
+    - https://docs.fortinet.com/document/fortimanager/7.2.1/log-message-reference/683345/introduction
+
+sources:
```

Comment:
Ditto Question-desing: Did you try to see if it goes through the queue-syslog decoder first? If it goes through this it is sure that it goes through the queue-localfile so I suppose so. Please add queue-syslog to the source

Reply:
Not apply for this decoder: decoder/queue-syslog/0  ->  failure